servers/list-api/src/businessEvents/unifiedEventKinesisHandler.ts

```diff
@@ -45,13 +45,15 @@ export class UnifiedEventKinesisHandler {
 
     try {
       await this.kinesis.send(putCommand);
-    } catch {
+    } catch (error) {
       serverLogger.error('Failed to send event(s) to kinesis stream', {
         stream: config.aws.kinesis.unifiedEvents.streamName,
         event: JSON.stringify(unifiedEvent),
+        error,
       });
       Sentry.addBreadcrumb({
         message: `Failed Events: \n ${JSON.stringify(unifiedEvent)}`,
+        data: { error: error },
```

Comment:
I think it would be more idiomatic to keep the message as a breadcrumb and change line 60 to be capturing the original error instead.